packages/shared/src/components/fields/Radio.tsx

```diff
@@ -39,7 +39,7 @@ export function Radio({
         <RadioItem
           key={option.value}
           name={name}
-          id={option.id || option.value}
+          id={`${name}-${option.id || option.value}`}
```

Comment:
Problem is when you have multiple `Radio` that render same options, like we have in `Details`. One for posting and second for invite permission. Each component renders two `RadioItem` groups that have duplicate ids with ones from other radio group. This is why I added `name`  so that even though options are the same we still have unique ids because each `Radio` does have unique name which is used in form's `onChange`.

Reply:
Oh I see :)
Cool, go ahead in that case

Reply:
Also see the commit message for 87786d9e7d78d227879414bbeb5aed0191b2714a